routers/web/user/profile.go

```diff
@@ -90,6 +91,29 @@ func Profile(ctx *context.Context) {
 		ctx.Data["RenderedDescription"] = content
 	}
 
+	// Fetches user's .profile/README.md and adds it to the users profile (if it exists)
+	resp, err := http.Get(setting.AppURL + ctx.ContextUser.Name + "/.profile/raw/branch/main/README.md")
```

Comment:
suggest get default branch instead of using `main` directly.

Reply:
Good idea, will fix